package.json

```diff
@@ -23,13 +23,14 @@
   },
   "dependencies": {
     "boom": "4.x.x",
-    "hoek": "4.1.0"
+    "hoek": "4.1.0",
+    "joi": "9.1.0"
```

Comment:
Can you use the caret version form for hoek and joi here? e.g. `"hoek": "^4.1.0"`.

After this then I think we can merge - great work @jonathansamines!